### PR TITLE
Adjust gas cost for Sapphire and Emerald

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

File: src/constant/paratimeConfig.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ const sapphire_config = {
     displayOrder: 2,
     defaultFees: {
         gasPrice: 100n,
-        feeGas: 15_000n,
+        feeGas: 61_290n,
```
Comment (buberdds) [inline src/constant/paratimeConfig.js:75]: @kostko is this a correct value we want to hardcode?

Comment (lukaw3d) [reply src/constant/paratimeConfig.js:75]: I would increase higher just in case, but I have no idea if that's necessary

Comment (lukaw3d) [reply src/constant/paratimeConfig.js:75]: with a high nonce, high fee, high amount I get a higher estimate: 61316
Round up to 70_000?
```js
// node a.mjs
import oasis from '@oasisprotocol/client' // @alpha version
import oasisRT from '@oasisprotocol/client-rt' // @alpha version
import xhr2 from 'xhr2'
global.XMLHttpRequest = xhr2

const sapphireConfig = {
  mainnet: {
    address: 'oasis1qrd3mnzhhgst26hsp96uf45yhq6zlax0cuzdgcfc',
    runtimeId: '000000000000000000000000000000000000000000000000f80306c9858e7279',
  },
  testnet: {
    address: 'oasis1qqczuf3x6glkgjuf0xgtcpjjw95r3crf7y2323xd',
    runtimeId: '000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c',
  },
}

const txWrapper = new oasisRT.consensusAccounts.Wrapper(oasis.misc.fromHex(sapphireConfig.mainnet.runtimeId)).callDeposit()
const feeAmount = 10n**18n
const feeGas = 10n**18n
const signerInfo = {
  address_spec: {
    signature: { ed25519: oasis.signature.NaclSigner.fromRandom('this key is not important').public() },
  },
  nonce: 10000n,
}

txWrapper
  .setBody({
    amount: [
      oasis.quantity.fromBigInt(10n**18n * 1000000000n),
      oasisRT.token.NATIVE_DENOMINATION,
    ],
    to: oasis.staking.addressFromBech32(
      // await getEvmBech32Address('0x0000000000000000000000000000000000000000')
      'oasis1qq2v39p9fqk997vk6742axrzqyu9v2ncyuqt8uek'
    ),
  })
  .setFeeAmount([oasis.quantity.fromBigInt(feeAmount), oasisRT.token.NATIVE_DENOMINATION])
  .setFeeGas(feeGas)
  .setFeeConsensusMessages(1)
  .setSignerInfo([signerInfo])

const estimatePriceResult = await new oasisRT.core.Wrapper(oasis.misc.fromHex(sapphireConfig.mainnet.runtimeId))
  .queryEstimateGas()
  .setArgs({ tx: txWrapper.transaction })
  .query(nic)

console.log('estimate deposit fee', estimatePriceResult)
```

Comment (kostko) [reply src/constant/paratimeConfig.js:75]: Yeah better round up a bit, 70k seems fair. Also note the same will apply to Emerald soon, so I would just adjust it now.

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**